zfs_zpool.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash -eu
```

Comment:
We're trying to standardize on `/usr/bin/env ...` for shebangs in this repo.
```suggestion
#!/usr/bin/env bash

 set -eu
```

Reply:
Done (and rebased on master)